flux_led/__main__.py

```diff
@@ -940,7 +947,29 @@ def setRgbw(self, r=None, g=None, b=None, w=None, persist=True,
                 msg.append(int(w))
             else:
                 msg.append(int(0))
-            msg.append(special)
+
+            if self.protocol == "LEDENET":
+                # LEDENET devices support two white outputs for cold and warm. We set
+                # the second one here - if we're only setting a single white value,
+                # we set the second output to be the same as the first
+                if w2 is not None:
```

Comment:
Oh, I should also add.. This is basically a mask for the controller. It allows you to set just the whites w/o having to send the colors. That's what the app does on its individual screens, it leaves the colors alone while changing the white, by masking out the colors.

If you send 0x00, then everything is updated, so if you don't include the colors you want set, then it'll overwrite them w/ 0, basically turning off the RGB LEDs. Don't think that would be intended..